templates/Pages/bof_isc21.php

```diff
@@ -0,0 +1,123 @@
+<?php $this->assign('title', 'Birds-of-a-Feather - ISC 21'); ?>
+
+<nav id="breadcrumb">
+    <p>YOU ARE HERE</p>
+
+    <?php
+    $this->Breadcrumbs->add(__('HOME'), ['controller' => 'submissions', 'action' => 'latest']);
+    $this->Breadcrumbs->add(__('BOF'), ['controller' => 'pages', 'action' => 'display', 'bofs']);
+    $this->Breadcrumbs->add(__("ISC'19"), ['controller' => 'pages', 'action' => 'display', 'bof-isc19']);
+
+    echo $this->Breadcrumbs->render([], ['separator' => ' / ']);
+    ?>
+</nav>
+
+<div class="content">
+    <h2>ISC'21</h2>
+
+    <h3>The IO500 and the Virtual Institute of I/O</h3>
+
+    <table class="tb">
+        <thread>
+            <th>
+                <td></td>
+                <td></td>
+            </th>
+        </thread>
+        <tbody>
+            <tr>
+                <td><strong>Date</strong></td>
+                <td>Tuesday, June 28th, 13:45-14:45</td>
+            </tr>
+            <tr>
+                <td><strong>Venue</strong></td>
+                <td>Virtual, details to be announced</td>
+            </tr>
+        </tbody>
+    </table>
+
+    <p>
+        <a class="link" href="https://www.isc-hpc.com/files/isc_events/documents/ISC2021-Digital/ISC2021_Program-at-a-Glance.pdf" target="_blank">Link to the official announcement</a>.
+    </p>
+
+    <h3>Abstract</h3>
+
+    <p>
+        The IO500 is quickly becoming the de facto benchmarking standard for
+        HPC storage. Developed two years ago, the IO500 has published three
+        official lists so far. A BoF highlight is the presentation of the
+        fourth IO500 list.
```

Comment:
this will be the eighth  list.

Reply:
Let's get rid of such a relative note as it is annoying to update every time. Maybe:
Since the first release at Supercomputing 2017, the IO500 publishes two official lists per year.